### DISCUSSION TITLE
Error installing templ

### DISCUSSION THREAD

**Discussion by AlejandroSoftwareDeveloper** [a-h/templ#514] [Q&A]
Hi I am getting this error when i try to install templ. I tried even with vpn but the error still continue.

go install github.com/a-h/templ/cmd/templ@latest
go: downloading github.com/segmentio/encoding v0.3.6
C:\Users\MyUser\go\pkg\mod\github.com\a-h\protocol@v0.0.0-20230224160810-b4eec67c1c22\base.go:9:2: github.com/segmentio/encoding@v0.3.6: reading https://proxy.golang.org/github.com/segmentio/encoding/@v/v0.3.6.zip: 403 Forbidden

**Comment by a-h** [comment]
I think this was "answered" in https://github.com/a-h/templ/issues/515

If you get stuck, hitting up the Gophers Slack channel is a good way to get help with general issues.
